pol.html

```diff
@@ -348,10 +348,9 @@ <h2 id="milestones">Set Conformance Milestones</h2>
 
 		<p class="totop"><a href="#contents">back to page contents</a></p>
 
-
 		<h2 id="thirdparty">Consider Third-party Content</h2>
```

Comment:
This got me thinking if we need a "mobile content and applications" section? Would be essentially very similar just explaining the relevance of mobile and that it is equally part of the website (just as third-party content is).